notebooks/101-tensorflow-to-openvino/README.md

```diff
@@ -0,0 +1,15 @@
+# TensorFlow to OpenVINO Model Conversion Tutorial
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F101-tensorflow-to-openvino%2F101-tensorflow-to-openvino.ipynb)
+
+![coco image](https://user-images.githubusercontent.com/15709723/127032784-8846df6f-0bfb-44ce-8920-76bcc0b5199e.jpg)
+
+This tutorial explains how to convert [TensorFlow](www.tensorflow.org) models to OpenVINO IR with FP16 precision. Inside the notebook we show how to convert the [TensorFlow MobilenetV3 model](https://github.com/tensorflow/models/tree/master/research/slim/nets/mobilenet) and then classify an image with OpenVINO's Inference Engine.
```

Comment:
```suggestion
This tutorial explains how to convert [TensorFlow](www.tensorflow.org) models to OpenVINO IR. Inside the notebook we show how to convert the [TensorFlow MobilenetV3 model](https://github.com/tensorflow/models/tree/master/research/slim/nets/mobilenet) and then classify an image with OpenVINO's Inference Engine.
```

Reply:
I suggest deleting the FP16 part because that is not the goal of the tutorial and we do not want people to think they need a different tutorial if they want to convert to FP32.